### PR TITLE
Updated Animation, Drawable, Events & Interactivity.

### PR DESCRIPTION
**Pull request by hristo-kanchev**
Description:

In this PR we now have:
- Fixed URLs that either point to dead links or to old outdated links
- Achieved better copy text so we are communicating clearly to the reader
- Removed unnecessary section - H2D-Interactive

### CODE REVIEW

File: Animation.md [hunk 1]
```diff
@@ -42,10 +42,4 @@ anim.onAnimEnd = function() {
...
 
-`Anim` instances have other properties which can be discovered by reviewing the [`h2d.Anim`](https://github.com/ncannasse/heaps/blob/master/h2d/Anim.hx) class.
-
-## Using image resources
-
-What you'll probably need for your game is actually using your image resources and sprites.
-- This is a small external github sample: https://github.com/Beeblerox/Simplest-Heaps-Examples/tree/master/04_heaps_anim
-It uses an image to create a sprite strip/sheet from it.
\ No newline at end of file
+`Anim` instances have other properties which can be discovered by reviewing the [`h2d.Anim`](https://github.com/HeapsIO/heaps/blob/master/h2d/Anim.hx) class.
```
[the remark this answers is not in the document]
Comment (Yanrishatum) [reply Animation.md:45]: Shouldn't it point at `https://heaps.io/api/h2d/Anim.html` for consistency with other class links?

[the remark this answers is not in the document]
Comment (hristo-kanchev) [reply Animation.md]: @Yanrishatum You are right. Just pushed a fix. Thanks for the review!

File: H2D-Interactive.md [hunk 2]
```diff
@@ -1,44 +0,0 @@
-# H2D Interactive
```
[the remark this answers is not in the document]
Comment (yuxiaomao) [reply H2D-Interactive.md:1]: Do you think it worth to be added in samples? If that's the case, the original author @marcelEuchnerMartinez can create a PR on heaps

[the remark this answers is not in the document]
Comment (hristo-kanchev) [reply H2D-Interactive.md:1]: I think it's not needed for now. We can always decide to re-add this after we do the first initial cleanup pass.

**Comment by nayata** [comment]
There is a problem with the links, regarding their location: in the wiki or on the site.

Example: https://heaps.io/documentation/objects.html

> Objects can be added to a [scene](https://heaps.io/Scenes) directly

In wiki this link works fine but not in the site. On the site this link turns into (https://heaps.io/Scenes)

**Comment by nayata** [comment]
Same problem link in https://heaps.io/documentation/h2d-introduction.html

> The [next section](https://heaps.io/Objects) will discuss h2d.Object a bit closer.

**Comment by hristo-kanchev** [comment]
Hey @nayata. I fixed these changes in this PR https://github.com/HeapsIO/heaps-doc/pull/102. Thanks for spotting those!

**Pull request merged**